ChangeHistory

```diff
@@ -6,6 +6,18 @@ Releases are tagged on the 'master' branch as "g4cmp-Vxx-yy-zz".
 Features and bug fixes are tagged on the 'develop' branch using the JIRA
 ticket identifier, "G4CMP-nnn", or using the GitHub pull request, "PR-nn".
 
+[ Modifications included on branch G4CMP-404, put subtask tags below it: ]
+2024-05-07  G4CMP-404 : Improve charge carriers-phonons scattering.
+2024-09-27  G4CMP-120 : IV scattering process should include kinematic effects.
+2024-09-27  G4CMP-429 : Add IV order and # final valleys to config.txt.
+
+[ Main line "develop" ChangeHistory; when merging, put develop updates here: ]
+
+2024-09-25  G4CMP-422 : Allow user to override IV rate model with UI command.
+
+2024-09-25  g4cmp-V09-01-00 Support sharing mesh fields across threads.
+2024-09-25  G4CMP-430 : Update CONTRIBUTING for ChangeHistory with big tasks.
+2024-09-25  G4CMP-244 : Reduce memory footprint of mesh electric field objects.
```

Comment:
I'm a little confused here, Emile.  Why do you have these lines in the ChangeHistory file for your branch?  I just checked the code, and you do *NOT* have any of the code related to these lines in your branch.  Did you hand-copy the ChangeHistory file?  Did you cherry-pick all of G4CMP-422?  Or did you just cherry-pick the last commit, which only had the ChangeHistory modification?

Reply:
I hand-copied the ChangeHistory file. I was not sure what you meant exactly by updating the ChangeHistory file (only updating the date?) so I wrongly assumed I needed to add the latest develop changes because of line 14 of ChangeHistory : [ Main line "develop" ChangeHistory; when merging, put develop updates here: ]

I have removed the changes. It makes more sense to only add the branches which are on G4CMP-429, sorry!

Reply:
Thanks, Emile.  I'll check it out.  The ChangeHistory file should always reflect exactly the state of the branch that it's on.  That's why you set the datestamp when you merge.  

If and only if you do something like "merge develop onto my branch", then you'll get all the new stuff from develop's ChangeHistory automatically, and you take care of organizing it during conflict resolution.

The easiest way to think about all this: if someone else (not you, or Future You a month from now) checks out your branch for the first time, what will they see?  And what will they _expect_ to see?  The ChangeHistory file helps them with that, and so it should reflect what they need to know.